src/main/java/org/fundacionjala/coding/cesar/FizzBuzz.java

```diff
@@ -0,0 +1,42 @@
+package org.fundacionjala.coding.cesar;
+
+import java.util.ArrayList;
+
+import java.util.stream.Collectors;
+
+/**
+ *
+ * @author admin-hp
+ */
+public class FizzBuzz {
+    private static final int FIVE = 5;
+    private static final int THREE = 3;
+    /**
+     * @param range List from int.
+     * @return list String replace.
+     */
+    public ArrayList<String> doBuzzFizz(final ArrayList<Integer> range) {
+        return range
+                .stream()
+                .map(i -> this.getRespectiveFizzBuzz(i))
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace this lambda with a method reference. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1612)